docs/_future/getting_started_future.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Getting started
+order: 1
+slug: getting-started-future
+lang: en
+layout: tutorial
+permalink: /tutorial/getting-started-future
+---
+
+## Getting started <span class="label-beta">BETA</span>
+
+<div class="section-content">
+This guide will cover how to get started with your next-gen platform using Bolt for Python by setting up the Slack CLI and also installing the required dependencies.
+</div>
+
+---
+
+### What is the next-generation platform? {#next-gen}
+
+You can find all the answers to this question on Slack's <a href="https://api.slack.com/future/intro" target="_blank">official introduction page.</a>
+
+<p class="alert alert_info"><ts-icon class="ts_icon_info_circle"></ts-icon>Our next-generation platform is currently in beta. <a href="https://api.slack.com/future/survey" target="_blank">Your feedback is most welcome.</a> - all feedback will help shape the future platform experience!</p>
+
+---
+
+### Setting up the CLI {#setting-up-cli}
+
+To build a next-generation app with Bolt for Python, you'll need to get the Slack CLI.
+
+**Install the Slack Cli** by following this [Quickstart Guide](https://api.slack.com/future/quickstart).
+
+---
+
+### Install dependencies {#install-dependencies}
+
+Once the CLI is set up, make sure your machine has the most recent version of [Python](https://www.python.org/) installed. you can either install Python through a package manager (such as [Homebrew](https://brew.sh/) for Mac) or directly from the website.
+
+---
+
+### Accept the Beta Terms of Service {#accept-tos}
+
+In order to use the next-generation platform features, you'll need to accept a Terms of Service for Slack Platform Beta. To do so, go to the Workspace Settings of the workspace you'll be developing with, then go to the Permissions tab. Find the "Slack Platform Beta" heading, then select the checkbox that says "I have read and agree to the Slack Platform Beta Service Terms" and save the section. Once you have accepted, the section will look like this:
```

Comment:
Let's just link to this page and cut this text down entirely wrt to instructions. https://api.slack.com/future/faq#beta-reqs

Reply:
Actually! Just found out that the Beta ToS is _only required_ to be signed if the developer is planning to deploy. SInce these apps won't be deployed to Slack infrastructure, we can remove the entire ToS section :)

Reply:
Ohh this is great, it will simplify the guide 💯